packages/web-components/src/components/ic-status-tag/ic-status-tag.tsx

```diff
@@ -44,14 +55,15 @@ export class StatusTag {
   }
 
   render() {
-    const { label, status, appearance, small, announced } = this;
+    const { label, status, appearance, variant, small, size, announced } = this;
     return (
       <Host role={announced ? "status" : null} aria-label="Status">
         <strong
           class={{
             ["tag"]: true,
-            [`${appearance}-${status}`]: true,
-            ["small"]: small,
+            [`${appearance}-${status}`]: appearance !== undefined,
```

Comment:
Could use `isPropDefined`?

Reply:
I don't think this can be done as isPropDefined doesn't return a boolean

Reply:
If it returns a value which isn't falsey, it should be okay

Reply:
I just tried updating it and it's complaining that type 'string | boolean' is not assignable to type 'boolean' :(